src/features/validators/useValidatorGroups.ts

```diff
@@ -112,10 +112,68 @@ async function fetchValidatorGroupInfo(publicClient: PublicClient) {
     group.eligible = true;
     group.capacity = getValidatorGroupCapacity(group, validatorAddrs.length, totalLocked);
   }
+  // note this modifies the groups object
+  await setVotesForIneligibleGroups(publicClient, groups);
```

Comment:
Since we wanna use groups as an array anyways, might as well not modify in place to avoid weird side-effects in the future?

`async function setVotesForIneligibleGroups(...): Promise<ValidatorGroup[]>` ?